src/modules/events/events.js

```diff
@@ -29,3 +30,11 @@ bus.on('change', (change) => {
 		else winston.info('Events registered');
 	});
 });
+
+bus.on('http/init', app => {
+	app.use(route.post('/x/change', function *() {
```

Comment:
@satya164 session module ensure that. All change events should have "auth" property for the session module to let it go forward. If not auth property then it should have a source as belong. If sessionID cannot always be sent then we should think of other ways.

Having it as part of change event and allowing it from a end point without any authentication cannot be allowed.